.circleci/config.yml

```diff
@@ -32,22 +32,7 @@ jobs:
         command: go env
     - run:
         name: Make
-        command: make
-    - run:
-        name: Verify vendoring
-        command: make vendorcheck
-    - run:
-        name: Run golangci-lint
-        command: make cross-lint
-    - run:
-        name: Make cross
-        command: make cross
-    - run:
-        name: Run unit tests
-        command: make test
-    - run:
-        name: Compile integration tests
-        command: make build_integration
+        command: make check
```

Comment:
I did not change circleci configuration on purpose as it was slightly more informative to have the target split. No strong feeling either way though.